apps/web/src/components/utils/matchers/markdown/MDCodeMatcher.tsx

```diff
@@ -13,8 +13,13 @@ export class MDCodeMatcher extends Matcher {
   }
 
   match(value: string) {
-    return this.doMatch(value, /`(.*?)`/u, (matches) => ({
-      match: matches[1]
-    }));
+    return this.doMatch(
+      value,
+      /`(.*?)`/u,
+      (matches) => ({
+        match: matches[1]
+      }),
+      true
+    );
```

Comment:
Any changes here? 🤔  I'm not seeing any 😕

Reply:
Might be hard to spot :) but I added the "true" at the end (which marks the formatter as "void").
And because of the added parameter, the Prettier formatter required all the extra line breaks.